src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --color: #00acdc;
+}
+
+.menu {
+  max-width: 1200px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.menu__logo {
+  display: inline-block;
+  padding: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.menu__container {
+  display: flex;
+  flex-direction: row;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+  color: black;
```

Comment:
1. More correctly provide color value in #hex or rgba format
2. If you already use variable for blue color, please also move this black color to variable, because if you need to change this 'default' color, it will be enough for you to change the value of the variable and not look for it through the whole code